### DISCUSSION TITLE
Unavailable repo - "this repository is temporary unavailable"

### DISCUSSION THREAD

**Discussion by dstumpy** [community/community#18143] [Projects and Issues]
I'm getting "this repository is temporarily unavailable. This repository is moving to a newer, better home. It’ll be back in just a few minutes." when I try to open some of my private repositories.

What's going on here, and how can this be fixed?

**Comment by Synceratus** [comment]
Well, 7 hours have passed. Check if it's back.

**Reply by dstumpy** [reply]
No, it is still unavailable.

**Reply by InaJaweed** [reply]
did it get fixed?

**Reply by dstumpy** [reply]
Unfortunately not.

**Comment by iurage** [comment]
Same problem here

**Comment by mandeep111** [comment]
I'm facing the same problem in one of my repositories.

**Comment by MartinLesterSynamedia** [comment]
As a **site admin**, you need to go to the admin section for the repo that is behaving like this.
Once there go to the admin tab, then the admin menu (on the left)
Find the `Single Repository Lock` section and click `Unlock`
Problem has "gone away".

Apparently the most likely culprit was we recent had to do a HA replica fail over, but that's not really very satisfying as some of the repos were created days after that was completed.

Edit: Even better, this issue was affecting all the repos in this 1 org, but unlocking just 1 seems to have fixed them all. Maybe I got lucky :)

**Comment by unmaykr-aftermath** [comment]
Same problem, it's been several hours since we tried to make one of our repositories public
